bridges/TheFarSideBridge.php

```diff
@@ -0,0 +1,52 @@
+<?php
+class TheFarSideBridge extends BridgeAbstract {
+	const NAME = 'The Far Side Bridge';
+	const URI = 'https://www.thefarside.com';
+	const DESCRIPTION = 'Returns the daily dose';
+	const MAINTAINER = 'VerifiedJoseph';
+	const PARAMETERS = array();
+
+	const CACHE_TIMEOUT = 3600; // 1 hour
+
+	public function collectData() {
+		$html = getSimpleHTMLDOM(self::URI)
+			or returnServerError('Could not request: ' . self::URI);
+
+		$item = array();
+		$item['uri'] = self::URI . date('/Y/m/d', strtotime($html->find('h3', 0)->innertext));
+		$item['title'] = $html->find('h3', 0)->innertext;
+		$item['timestamp'] = $html->find('h3', 0)->innertext;
```

Comment:
Returns null or false. Suggesting the same as above.

Reply:
Fix by https://github.com/RSS-Bridge/rss-bridge/pull/1484/commits/03cf13845bd58892a86e50fbd566b615841f7d65